openslides/mediafiles/static/templates/mediafiles/mediafile-list.html

```diff
@@ -63,9 +63,63 @@ <h1 translate>Files</h1>
               </button>
             </div>
           </nav>
+          <nav ng-show="getType(presentedMediafile) === 'image'" ng-class="getNavStyle(scroll)" class="form-inline">
+            <div class="btn-group">
+              <button class="btn btn-default" ng-click="mediafileToggleFullscreen()" title="{{ 'Toggle fullscreen' | translate }}"
+				ng-class="presentedMediafile.fullscreen ? 'btn-primary' : 'btn-default'">
```

Comment:
Fix indentation here (use 2x 4 spaces for this line, beginning from `<button ...`.